crawlers/mooncrawl/sample.env

```diff
@@ -31,3 +31,8 @@ export MOONSTREAM_S3_QUERIES_BUCKET_PREFIX="dev"
 # 3rd parties environment variables
 export MOONSTREAM_ETHERSCAN_TOKEN="<Token_for_etherscan>"
 export COINMARKETCAP_API_KEY="<API_key_to_parse_conmarketcap>"
+
+
+# Custom crawler
+export CUSTOM_CRAWLER_S3_BUCKET="<public_bucket>"
```

Comment:
Pls use naming 
`MOONSTREAM_S3_DATA_BUCKET` and `MOONSTREAM_S3_DATA_BUCKET_PREFIX`